client/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,162 @@
+import { AppBar, Toolbar, Typography, Badge, IconButton, Drawer, Box, Grid } from '@material-ui/core';
+
+import { Link } from 'react-router-dom';
+import MenuIcon from '@material-ui/icons/Menu';
+import React, { useState, useEffect } from 'react';
+import AvatarDisplay from '../AvatarDisplay/AvatarDisplay';
+import useStyles from './useStyles';
+import PetsIcon from '@material-ui/icons/Pets';
+import { User } from '../../interface/User';
+
+interface Props {
+  logOut: string;
+  profile: string;
+  mySitter: string;
+  becomeSitter: string;
+  messages: string;
+  loggedIn: boolean;
+  loggedInUser: User;
+  open: boolean;
+}
+
+const Navbar = ({
+  logOut,
+  profile,
+  mySitter,
+  becomeSitter,
+  messages,
+  loggedInUser,
+  loggedIn,
+  open,
+}: Props): JSX.Element => {
+  const classes = useStyles();
+
+  const [state, setState] = useState({
+    mobileView: false,
+    drawerOpen: false,
+  });
+
+  const { mobileView, drawerOpen } = state;
+
+  useEffect(() => {
+    const setResponsiveness = () => {
+      return window.innerWidth < 1200
+        ? setState((prevState) => ({ ...prevState, mobileView: true }))
+        : setState((prevState) => ({ ...prevState, mobileView: false }));
+    };
+
+    setResponsiveness();
+
+    window.addEventListener('resize', () => setResponsiveness());
+
+    return () => {
+      window.removeEventListener('resize', () => setResponsiveness());
+    };
+  }, []);
+
+  const displayDesktop = () => {
+    return (
+      <Toolbar>
+        <Grid container>
+          <Grid item container sm={2} alignContent="center">
+            <PetsIcon fontSize="large" className={classes.icon} />
+            <Typography variant="h5" align="center" noWrap className={classes.logo}>
+              LovingSitter.
+            </Typography>
+          </Grid>
+          <Grid sm={10} item container className={classes.toolbarGrid}>
+            {getMenuButtons()}
+          </Grid>
+        </Grid>
+      </Toolbar>
+    );
+  };
```

Comment:
We probably don't need to define this inside of the `Navbar` component - it's better to separate each of these functions into their own components. If you want to you, can pass `className`s (as well as other props) to these in order avoid re-using the `useStyles` hook within each component too. No need to separate the files though!